projects/plugins/protect/src/js/components/summary/index.jsx

```diff
@@ -38,7 +41,7 @@ const Summary = () => {
 						{ sprintf(
 							/* translators: %s: Latest check date  */
 							__( 'Latest results as of %s', 'jetpack-protect' ),
-							dateI18n( 'F jS', lastChecked )
+							dateI18n( 'F jS', lastCheckedLocalTimestamp )
```

Comment:
The [dateI18n](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-date/#datei18n) function actually accepts a third argument to specify the timezone or UTC offset :+1:

Reply:
If we supply this third argument, it appears the internal mechanism that converts it to the timezone specified in the WordPress installation settings is not used.